samples/common/Kconfig

```diff
@@ -80,7 +105,7 @@ config GOLIOTH_SAMPLE_WIFI
 
 config GOLIOTH_SAMPLE_WIFI_SETTINGS
 	bool "Load SSID and PSK from settigs subsystem"
-	default y if GOLIOTH_SYSTEM_SETTINGS
+	default n
```

Comment:
`default n` is implicit, so no need to set that (this line can be removed). Also it is quite hard to which logical change this is related, since all changes are in single commit.